extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -700,12 +711,7 @@ export class SubscriptionService {
    * @private
    */
   isPageUnlocked_() {
-    return (
-      !this.pageConfig_.isLocked() ||
-      // If a service marks `alwaysGrant` as true, it will unlock the page
-      // unless the viewer provides auth.
-      (this.platformConfig_['alwaysGrant'] && !this.doesViewerProvideAuth_)
```

Comment:
Seems safe to remove this check for `!this.doesViewerProvideAuth_`. This way free pages will act consistently, whether they are marked as free in the page or platform config

Reply:
Check with news team that this is desired behavior.